### PR TITLE
[Benchmark] Add BiSeNetV1 COCO-Stuff 164k benchmark

### PR DESCRIPTION
**Pull request by MengzhangLI**
Uploading BiSeNetV1 model (R18, R50, R101) on Coco-Stuff 164k dataset.

Paper: https://arxiv.org/pdf/1808.00897.pdf
R18: 28.1(paper), 28.55(ours)
R101: 31.3(paper), 37.38(ours)

image

### CODE REVIEW

File: configs/bisenetv1/bisenetv1_r50-d32_lr5e3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+_base_ = './bisenetv1_r50-d32_lr5e3_4x4_512x512_160k_coco-stuff164k.py'
```
Comment (Junjun2016) [inline configs/bisenetv1/bisenetv1_r50-d32_lr5e3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py:1]: Should check the config name, please refer to our config name manual.
It seems that the lr position is not correct.
5e3 --> 5e-3?

File: configs/bisenetv1/bisenetv1_r101-d32_lr5e3_4x4_512x512_160k_coco-stuff164k.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            in_channels=512,
+            channels=256,
+            num_convs=1,
+            num_classes=171,
+            in_index=2,
+            norm_cfg=norm_cfg,
+            concat_input=False),
+    ])
+lr_config = dict(warmup='linear', warmup_iters=1000)
+optimizer = dict(type='SGD', lr=0.005, momentum=0.9, weight_decay=0.0005)
```
Comment (Junjun2016) [inline configs/bisenetv1/bisenetv1_r101-d32_lr5e3_4x4_512x512_160k_coco-stuff164k.py:43]: Only the keys that are different from `base` should be included.

File: configs/bisenetv1/bisenetv1_r50-d32_lr5e3_4x4_512x512_160k_coco-stuff164k.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            in_channels=512,
+            channels=256,
+            num_convs=1,
+            num_classes=171,
+            in_index=2,
+            norm_cfg=norm_cfg,
+            concat_input=False),
+    ])
+lr_config = dict(warmup='linear', warmup_iters=1000)
+optimizer = dict(type='SGD', lr=0.005, momentum=0.9, weight_decay=0.0005)
```
Comment (Junjun2016) [inline configs/bisenetv1/bisenetv1_r50-d32_lr5e3_4x4_512x512_160k_coco-stuff164k.py:42]: Only the keys that are different from `base` should be included.

File: configs/bisenetv1/bisenetv1_r101-d32_lr5e-3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+_base_ = './bisenetv1_r101-d32_lr5e-3_4x4_512x512_160k_coco-stuff164k.py'
```
Comment (Junjun2016) [inline configs/bisenetv1/bisenetv1_r101-d32_lr5e-3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py:1]: pretrain should before lr.
Please refer to the config name manual and rename it.

File: configs/bisenetv1/bisenetv1_r18-d32_lr5e-3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+_base_ = './bisenetv1_r18-d32_lr5e-3_4x4_512x512_160k_coco-stuff164k.py'
```
Comment (Junjun2016) [inline configs/bisenetv1/bisenetv1_r18-d32_lr5e-3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py:1]: pretrain should before lr.
Please refer to the config name manual and rename it.

File: configs/bisenetv1/bisenetv1_r50-d32_lr5e-3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+_base_ = './bisenetv1_r50-d32_lr5e-3_4x4_512x512_160k_coco-stuff164k.py'
```
Comment (Junjun2016) [inline configs/bisenetv1/bisenetv1_r50-d32_lr5e-3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py:1]: pretrain should before lr.
Please refer to the config name manual and rename it.

File: configs/bisenetv1/bisenetv1_r50-d32_lr5e-3_4x4_512x512_160k_coco-stuff164k.py [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+            concat_input=False),
+        dict(
+            type='FCNHead',
+            in_channels=512,
+            channels=256,
+            num_convs=1,
+            num_classes=171,
+            in_index=2,
+            norm_cfg=norm_cfg,
+            concat_input=False),
```
Comment (Junjun2016) [inline configs/bisenetv1/bisenetv1_r50-d32_lr5e-3_4x4_512x512_160k_coco-stuff164k.py:39]: Could remove redundant keys from `_base_` file `../_base_/models/bisenetv1_r18-d32.py`.

**Comment by RockeyCoss** [comment]
Please merge the master branch into your branch, thank you.

**Review by Junjun2016 — APPROVED** [review]

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**